common/scala/src/whisk/core/entity/ActivationResult.scala

```diff
@@ -51,10 +50,10 @@ protected[core] object ActivationResponse extends DefaultJsonProtocol {
     /* The field name that is universally recognized as the marker of an error, from the application or otherwise. */
     val ERROR_FIELD: String = "error"
 
-    private val Success          = 0
-    private val ApplicationError = 1
-    private val ContainerError   = 2
-    private val WhiskError       = 3
+    val Success = 0
```

Comment:
nit pick - realign? i know it's annoying because i couldn't get auto format to todo the right thing,